six_dof_arm:
  hardware_interface:
     loop_hz: 10 # hz
     joints:
       - shoulder_pan_joint
       - shoulder_pitch_joint
       - elbow_pitch_joint
       - wrist_pitch_joint
       - gripper_roll_joint
       - finger_joint1
       - finger_joint2
